docs/conf.py

```diff
@@ -126,7 +126,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+# html_static_path = ['_static']
```

Comment:
commented out code?

Reply:
I can remove it. Should I remove the preceding paragraph too?

Reply:
(The "Add any paths" explanation)

Reply:
Went ahead and did it

Reply:
yes pls